include/pstl_offload/oneapi/dpl/dynamic_selection

```diff
@@ -0,0 +1,25 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) 2023 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef _ONEDPL_PSTL_OFFLOAD_TOP_LEVEL_ONEAPI_DPL_DYNAMIC_SELECTION
+
+#ifndef _ONEDPL_PSTL_OFFLOAD_TOP_LEVEL
+#    define _ONEDPL_PSTL_OFFLOAD_TOP_LEVEL_ONEAPI_DPL_DYNAMIC_SELECTION
+#    define _ONEDPL_PSTL_OFFLOAD_TOP_LEVEL
+#endif // _ONEDPL_PSTL_OFFLOAD_TOP_LEVEL
+
+#include_next <oneapi/dpl/dynamic_selection>
```

Comment:
Have you checked it with script? if not do it, please.

Reply:
The script did not report any issues.